pulp_container/app/settings.py

```diff
@@ -2,3 +2,41 @@
     "dynaconf_merge_unique": True,
     "reusable_conditions": ["pulp_container.app.global_access_conditions"],
 }
+
+ADDITIONAL_OCI_ARTIFACT_TYPES = {
```

Comment:
The question is whether we should move these values (ADDITIONAL_OCI_ARTIFACT_TYPES) to `constants.py` directly or not. I find the registration pretty redundant at this moment. Also, should we support `application/vnd.oras.config.v1+json` and `application/vnd.module.wasm.config.v1+json`?

Reply:
The registration is crucial because it gives flexibility to the admin to add/remove supported oci mediatyes as he deems necessary. We will not be able to support all of them out of the box, or keep up with the list - everyday some new comes up. In addition admin also would want to decide whether he needs to support al lthe list, or for example allow just helm.
If this is going to be moved to constants.py it will be static list, or you had something else in your mind?

> application/vnd.oras.config.v1+json and application/vnd.module.wasm.config.v1+json

This is up for discussion, my preference would be to stick to well-known types or types we know our stakeholders/users rely on and already use.

Reply:
I am totally okay with moving most of the entries (e.g., for helm charts and cosign) to `constants.py`. We already support these media types, do not we? I found the registration of these particular media types to be redundant. I am fine with leaving the registration feature in place to allow some administering.